workflow-init/profile/nab.yml

```diff
@@ -235,3 +235,16 @@ component:
       - *backend_rest
       - *backend_grouping
       - *backend_search
+
+  configuration:
+    parameters:
+      <<: *defaultParameters
+      kustomize: bitbucket-fake
+      java-version: 17
+    ecr:
+      <<: *defaultECR
+      repository: hawk-configuration
+      repository-prefix: hawk-
```

Comment:
repository-prefix is used sometimes when jib argument meaning is different for some repositories

it should match prefix of image repository